packages/react-devtools-timeline/src/types.js

```diff
@@ -51,6 +51,7 @@ export type ReactScheduleRenderEvent = {|
 |};
 export type ReactScheduleStateUpdateEvent = {|
   ...BaseReactScheduleEvent,
+  +componentStack?: string,
```

Comment:
Shouldn't this field be on the `ReactScheduleStateUpdateEvent` only (for now)? Something like?

```js
export type ReactScheduleStateUpdateEvent = {|
  ...BaseReactScheduleEvent,
  +componentStack: string | null,
  +type: 'schedule-state-update',
|};
```

Reply:
Could you clarify what you mean? This field _is_ only on the `ReactScheduleStateUpdateEvent`. Do you mean that you'd prefer componentStack to always explicitly defined on the `ReactScheduleStateUpdateEvent`?

Reply:
Oops. I misread this as a new _optional_ field to `BaseReactScheduleEvent`. I was just saying we should add it to only the `ReactScheduleStateUpdateEvent` type (which you already did). Ignore me.